plugins/woocommerce/client/blocks/tests/e2e/tests/product-collection/inspector-controls.block_theme.spec.ts

```diff
@@ -602,10 +604,12 @@ test.describe( 'Product Collection: Inspector Controls', () => {
 				'Music',
 			] );
 
-			await page.getByLabel( 'Toggle block inserter' ).click();
-			await page.getByRole( 'tab', { name: 'Patterns' } ).click();
+			const addBlockButton = editor.canvas
+				.locator( '[data-type="woocommerce/product-collection"]' )
+				.getByRole( 'button', { name: 'Add block' } );
+			await addBlockButton.click();
 			await page.getByPlaceholder( 'Search' ).fill( 'product filters' );
-			await page.getByLabel( 'Product Filters' ).click();
+			await page.getByText( 'Product Filters' ).click();
```

Comment:
Same here. :slightly_smiling_face:

Reply:
Good idea, thanks for the suggestion. I pushed up changes.